pkg/didcomm/protocol/didexchange/states.go

```diff
@@ -43,6 +53,21 @@ func stateFromName(name string) (state, error) {
 	return nil, fmt.Errorf("invalid state name %s", name)
 }
 
+type noOp struct {
+}
+
+func (s *noOp) Name() string {
+	return "noop"
+}
+
+func (s *noOp) CanTransitionTo(_ state) bool {
+	return false
+}
+
+func (s *noOp) Execute(_ dispatcher.DIDCommMsg) (state, error) {
+	return nil, nil
```

Comment:
i would suggest error should be returned in this case.

Reply:
done